services/files/run_pull.go

```diff
@@ -171,6 +166,13 @@ func (s service) downloadFileTranslations(params PullParams, file sdkfile.File)
 			return err
 		}
 
+		if percents > 0 {
+			if complete < percents {
+				percentByExcludedFile[path] = strconv.Itoa(int(complete))
```

Comment:
Why not simply call here `fmt.Printf("skipped...`. I don't understand how this construction works. But it is totally fine to mix `downloaded` and `skipped` messages.